lib/Target/Jaxite/JaxiteEmitter.h

```diff
@@ -54,6 +55,9 @@ class JaxiteEmitter {
   LogicalResult printOperation(ConstantOp op);
   LogicalResult printOperation(tensor::ExtractOp op);
   LogicalResult printOperation(tensor::FromElementsOp op);
+  LogicalResult printOperation(memref::AllocOp op);
```

Comment:
are there any test cases covering the new memref support?

Reply:
Added tests to emit_jaxite.mlir